lib/daru/dataframe.rb

```diff
@@ -705,6 +705,14 @@ def rolling_fillna(direction=:forward)
       dup.rolling_fillna!(direction)
     end
 
+    # Return unique rows by vector specified or all vectors
```

Comment:
Docs should be more detailed, with some examples and description how uniqueness is considered. And param and return value descriptions.